scanners/boostsecurityio/native-scanner/rules.yaml

```diff
@@ -1508,6 +1509,42 @@ rules:
     name: cicd-scm-gh-org-insecure-webhook
     pretty_name: CI/CD - Insecure GitHub Webhooks
     ref: '{BOOSTSEC_DOC_BASE_URL}/rules/cicd-scm-gh-org-insecure-webhook.html'
+  cicd-scm-gh-org-number-of-owners:
+    categories:
+    - ALL
+    - supply-chain
+    - supply-chain-scm-weak-configuration
+    - supply-chain-cicd-weak-configuration
+    - boost-hardened
+    description: Checks for the number of GitHub Organization owners
+    group: supply-chain-scm-weak-configuration
+    name: cicd-scm-gh-org-number-of-owners
+    pretty_name: CI/CD - Invalid Number of GitHub Organization Owners
+    ref: '{BOOSTSEC_DOC_BASE_URL}/rules/cicd-scm-gh-org-number-of-owners.html'
+  cicd-scm-gh-repo-number-of-admins:
+    categories:
+    - ALL
+    - supply-chain
+    - supply-chain-scm-weak-configuration
+    - supply-chain-cicd-weak-configuration
+    - boost-hardened
+    description: Checks for the number of GitHub Repository contributors with administrative privileges.
+    group: supply-chain-scm-weak-configuration
+    name: cicd-scm-gh-repo-number-of-admins
+    pretty_name: CI/CD - Invalid Number of GitHub Repository Admins
```

Comment:
@Becojo hmm `Invalid Number` not sure that's the best way to put it

Maybe `Too Many GitHub Organization Owners` ?

Reply:
I don't love using "invalid" either.  It's really meant as "out of compliance with the established policy". The check can also be used to ensure there are a minimum number of owners so "too many owners" is not appropriate.

Reply:
@Becojo ok, well that's just a label we can always change ...